lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -145,7 +145,7 @@ def _handle_dropped_request(self, result, d, function_name):
 
     def render(self, request):
         time_in = utils.now()
-        assert self._check_headers(request), InvalidHeaderError
+        # assert self._check_headers(request), InvalidHeaderError
```

Comment:
While I agree that using authentication would be a good step forward, I don't think we should delay the electron launch due to that.

Also Electron is sending requests with essentially an empty origin, and no referrer.

![image](https://cloud.githubusercontent.com/assets/2380127/23032949/dcab42f4-f43b-11e6-8d05-365e629c7107.png)

I am not convinced checking headers buys us much in the way of security.  I'm open to being convinced otherwise.